features/feature_case/driverStreamAndEvent/driverStreamQuery.cu

```diff
@@ -0,0 +1,27 @@
+#include <cuda.h>
+#include <iostream>
```

Comment:
how this test is triggerred to run?

Reply:
`driverStreamAndEvent` is already a part of global variable `exec_tests`. The test file is added to _TEMPLATE_driverStreamEvemt_ now.